docs/source/license.rst

```diff
@@ -0,0 +1,16 @@
+License
+=======
```

Comment:
Good point-- I'll remove the copy of the license text in the docs and just direct that front page link to the license in the root dir.